phoenix-core/src/it/java/org/apache/phoenix/end2end/ExpFunctionEnd2EndIT.java

```diff
@@ -34,21 +34,26 @@
 /**
  * End to end tests for {@link ExpFunction}
  */
-public class ExpFunctionEnd2EndIT extends BaseHBaseManagedTimeIT {
+public class ExpFunctionEnd2EndIT extends BaseHBaseManagedTimeTableReuseIT {
 
     private static final String KEY = "key";
     private static final double ZERO = 1e-8;
+    private String signedTableName;
```

Comment:
Same comment as before. Don't have table names as member variables as it makes running tests in parallel difficult.